app/src/main/java/org/oppia/app/onboarding/OnboardingFragmentPresenter.kt

```diff
@@ -0,0 +1,114 @@
+package org.oppia.app.onboarding
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.LinearLayout
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.viewpager.widget.ViewPager
+import kotlinx.android.synthetic.main.onboarding_fragment.view.*
+import org.oppia.app.R
+import org.oppia.app.databinding.OnboardingFragmentBinding
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.viewmodel.ViewModelProvider
+import javax.inject.Inject
+
+/** The presenter for [OnboardingFragment]. */
+@FragmentScope
+class OnboardingFragmentPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val fragment: Fragment,
+  private val viewModelProvider: ViewModelProvider<OnboardingViewModel>
+) {
+  private val dotsList = ArrayList<ImageView>()
+  private lateinit var onboardingPagerAdapter: OnboardingPagerAdapter
+  private val routeToProfileListener = activity as RouteToProfileListener
+  private lateinit var binding: OnboardingFragmentBinding
+  private lateinit var slidesViewPager: ViewPager
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
+    binding = OnboardingFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+    // NB: Both the view model and lifecycle owner must be set in order to correctly bind LiveData elements to
+    // data-bound view models.
+    binding.let {
+      it.lifecycleOwner = fragment
+      it.presenter = this
+      it.viewModel = getOnboardingViewModel()
+    }
+    slidesViewPager = binding.root.onboarding_slide_view_pager as ViewPager
```

Comment:
It seems a bit cleaner to use the direct access variables, eg ``binding.onboardingSlideViewPager``. This avoids needing the synthetic imports.

Reply:
Done. removed `slidesViewPager`